### PR TITLE
Ancient markets page

### PR DESCRIPTION
**Pull request by NcyRocks**
Adds an ancient markets page at manifold.markets/ancient. Also adds links to most quests to make them easier to complete.

### CODE REVIEW

File: web/components/quests-or-streak.tsx [hunk 1]
```diff
@@ -201,7 +207,7 @@ const QuestRow = (props: {
           </span>
           <Col>
             <span className={clsx('sm:text-xl')}>
-              {title}
+              {link ? <a href={url}>{title}</a> : title}
```
Comment (jahooma) [inline web/components/quests-or-streak.tsx:210]: We use NextJS's link components. You could do this like:

`<Link href={href}>{title}</Link>`

File: web/components/quests-or-streak.tsx [hunk 2]
```diff
@@ -190,8 +194,10 @@ const QuestRow = (props: {
   status: string
   reward: number
   info?: string
+  link?: string
```
Comment (jahooma) [inline web/components/quests-or-streak.tsx:197]: Prefer the name `href` over `link` (it is not a full link)

**Review by jahooma — APPROVED** [review]
Awesome, seems pretty useful! Thanks Nathan!

File: web/components/quests-or-streak.tsx (hunk 1 above)
Comment (NcyRocks) [reply web/components/quests-or-streak.tsx]: Makes sense, fixed

File: web/components/quests-or-streak.tsx (hunk 2 above)
Comment (NcyRocks) [reply web/components/quests-or-streak.tsx]: Renamed, thanks!

**Pull request merged**